src/e2ir.c

```diff
@@ -1384,7 +1385,13 @@ elem *toElem(Expression *e, IRState *irs)
                 else
                 {
                     Symbol *csym = toSymbol(cd);
-                    ex = el_bin(OPcall,TYnptr,el_var(rtlsym[RTLSYM_NEWCLASS]),el_ptr(csym));
+                    if (global.params.tracegc && ne->loc.filename)
```

Comment:
Sometimes internally generated code doesn't have a location that would make any sense.